RPIPs/RPIP-46.md

```diff
@@ -28,42 +28,47 @@ A fixed percentage of RPL inflation is currently being used to fund ongoing main
 
 ## Specification
-1. Inflation settings SHALL be modified to retain inflation to the DAOs and eliminate inflation to NOs
-   1. `rpl.inflation.interval.rate` SHALL be set to `1000040763630249500` (1.5% per year)
-   2. Node Operators (`rocketClaimNode`) allocation SHALL be set to 0%
-   3. pDAO (`rocketClaimDAO`) allocation SHALL be set to 95%
-   4. oDAO (`rocketClaimTrustedNode`) allocation SHALL be set to 5%
-2. There SHALL be the following defined shares with settings: `node_operator_commission_share`, `voter_share`, `surplus_share`
-   1. `node_operator_commission_share`: each NO receives this percentage of the commission from the borrowed ETH on validators they run. Unlike the remainder of the shares, this is _not_ a protocol revenue (ie, it is not socialized).
+### UARS
+1. There SHALL be the following defined shares with settings: `node_operator_commission_share`, `node_operator_commission_share_council_adder`, `voter_share`, `surplus_share`
+   1. `node_operator_commission_share + node_operator_commission_share_council_adder`: each NO receives this percentage of the commission from the borrowed ETH on validators they run. Unlike the remainder of the shares, this is _not_ a protocol revenue (ie, it is not socialized).
    2. `voter_share`: each NO receives a share of revenue based on the vote-eligible RPL staked to their megapool. The overall voter share of revenue is based on the setting, and each NO receives a proportion of that based on `vote_eligible_RPL_in_their_megapool/total_vote_eligible_RPL_in_megapools`.
    3. `surplus_share`: this share of revenue is used to distribute revenue beyond that used for protocol operation (such as the shares above)
-3. `reth_commission` SHALL be defined as the sum of all defined shares that have settings
-4. `reth_share` SHALL be defined as `100% - reth_commission`
-5. Distributions of revenue from borrowed ETH MUST respect the defined shares
+2. `reth_commission` SHALL be defined as the sum of all defined shares that have settings
+3. `reth_share` SHALL be defined as `100% - reth_commission`
+4. Distributions of revenue from borrowed ETH MUST respect the defined shares
    1. If shares change between claims, distributions MUST make an effort to account for the different values. For example, a distribution could use a duration-weighted average share. Approximations MAY be used where they significantly reduce complexity and/or costs.
    2. Legacy minipools are an exception and SHALL continue to support earlier distribution methodologies 
-6. `node_operator_commission_share`, `surplus_share`, `increase_no_share_seal_increment`, `increase_no_share_seal_count`, and `allowlisted_controllers` SHALL be updateable by pDAO vote
-7. `node_operator_commission_share` and `surplus_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
+5. `node_operator_commission_share` and `surplus_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
```

Comment:
Should probably include `node_operator_commission_share_council_adder` in here as well.  This would allow the pDAO to increment NO commission and decrease the adder simultaneously, allowing council more room without changing the total NO commission.